### ISSUE TITLE
Testing test262 in different (web platform) agents

### ISSUE THREAD

**Issue by annevk** [web-platform-tests/rfcs#230]
Over in https://github.com/dpino/gecko-dev @dpino is working on ensuring that various `SharedArrayBuffer` tests from test262 run across the various agents defined by the web platform. A follow-up goal is to run all test262 tests across the various agents to ensure there's no weird bugs in the various JavaScript engines.

The idea is to host this "wrapper test suite" in web-platform-tests so all user agents can benefit.

If anyone has thoughts, ideas, or concerns that'd be great to hear.

cc @jgraham @domenic @foolip @ljharb @leobalter 

(Corresponding issue: https://github.com/dpino/gecko-dev/issues/21.)

**Comment by gsnedders** [comment]
Fixed via https://github.com/web-platform-tests/rfcs/pull/229.

**Comment by annevk** [comment]
I don’t think that fixed this? Reading the proposal it seems this was considered and wontfixed due to the amount of CI compute required.
